backend/v3/api/instance/v2/convert/instance.go

```diff
@@ -0,0 +1,51 @@
+package convert
+
+import (
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/zitadel/zitadel/backend/v3/domain"
+	instance "github.com/zitadel/zitadel/pkg/grpc/instance/v2beta"
+)
+
+/*
+ * Domain Model to GRPC v2
+ */
+
+func DomainInstanceListModelToGRPCResponse(instances []*domain.Instance) []*instance.Instance {
+	toReturn := make([]*instance.Instance, len(instances))
+
+	for i, inst := range instances {
+		toReturn[i] = DomainInstanceModelToGRPCResponse(inst)
+	}
+
+	return toReturn
+}
+
+func DomainInstanceModelToGRPCResponse(inst *domain.Instance) *instance.Instance {
+	return &instance.Instance{
+		Id:           inst.ID,
+		ChangeDate:   timestamppb.New(inst.UpdatedAt),
+		CreationDate: timestamppb.New(inst.CreatedAt),
+		State:        instance.State_STATE_RUNNING, // TODO(IAM-Marco): Not sure what to put here
+		Name:         inst.Name,
+		Version:      "",
```

Comment:
Stole the version from `internal/api/grpc/instance`
```suggestion
		Version:      build.Version(),
```